todo.js

```diff
@@ -15,3 +15,68 @@ Keep the responsibilities separated as best you can:
 4. Displaying information to the user
 5. Rather user input and taking the appropriate actions
 */
+
+const fs = require('fs');
+
+let ask = process.argv[2];
+let element = process.argv[3];
+// console.log(ask);
+// console.log(element);
+
+//var read = fs.readFileSync(user, 'utf-8');
+
+
+var read = fs.readFileSync('todos.txt', 'utf-8');
+
+let split = read.split(/[\r\n]+/g);
+let space = split.pop();
+//console.log(split);
+
+function list(array) {
+  for (let i = 0; i < array.length; i++) {
+    array[i] =`${i + 1}. [ ] ${array[i]}`;
+  }
+  return array;
+}
+function string(array) {
+  for (let i = 0; i < split.length; i++) {
+    console.log(split[i]);
+  }
+}
+
+function add(todo) {
+  split.push(element);
+
+  return string(split);
+}
+
+function remove(num) {
+  split.splice(num - 1,1);
+  return string(split);
+}
+
+function complete(array, num) {
+
+  for (let i = 0; i < array.length; i++) {
+    if (i === num-1) {
+    let oldArray = array[i];
+    let itemForDisplays = list(array);
+    array[i] =`${i + 1}. [X] ${oldArray}`;
+  }
+  //console.log(array);
+  return array;
+}
+}
+
+if (ask === 'list') {
```

Comment:
I recommend working on one feature at a time. Start with `list` and get it working completely.

Then do, say, `add` and get it working completely. "Completely" would involve also writing it to a file.

For example, you could have a function `saveTodoList('./todos.txt', todoList)` that takes a TODO list as input and writes the appropriate contents to the file `./todos.txt`.

Then any modification you perform in code can be saved to disk.